src/coreclr/nativeaot/Runtime/unix/UnixContext.cpp

```diff
@@ -10,38 +10,8 @@
 #include "regdisplay.h"
 #include "config.h"
 
-#include <libunwind.h>
-
 #include "UnixContext.h"
 
-// WebAssembly has a slightly different version of LibUnwind that doesn't define unw_get_save_loc
-#if defined(HOST_WASM)
```

Comment:
We've been avoiding deleting "harmless" WASM ifdefs to limit the number of merge conflicts in runtimelab. I haven't actually checked with @yowl or @SingleAccretion on how painful removing these would be so pinging them for thoughts on this. It is odd to have things that don't build here because we can't ensure it keeps building but also it doesn't hurt too much and my expectation is that WASM will come here eventually/some day.

Reply:
The code that calls `unw_get_save_loc` is being deleted below. This change is not actively harmful for wasm.